Code/PgSQL/rdkit/CMakeLists.txt

```diff
@@ -50,6 +50,7 @@ endif(NOT DEFINED PostgreSQL_CONFIG_DIR)
 if(NOT DEFINED PostgreSQL_CONFIG)
   set(PostgreSQL_CONFIG "${PostgreSQL_CONFIG_DIR}/pg_config")
 endif(NOT DEFINED PostgreSQL_CONFIG)
+set(PostgreSQL_CONFIG /usr/bin/pg_config)

```

Comment:
seems like part of a different PR

Reply:
Good catch. It looks like a locally modified file was committed accidentally with the rest of the PR.

Reply:
Right you are. Good catch. I will remove it